oximeter/impl/benches/quantile.rs

```diff
@@ -0,0 +1,53 @@
+use criterion::{criterion_group, criterion_main, BenchmarkId, Criterion};
+use oximeter_impl::Quantile;
+use rand_distr::{Distribution, Normal};
+
+/// Emulates baseline code in a Python implementation of the P²
+/// algorithm:
+/// <https://github.com/rfrenoy/psquare/blob/master/tests/test_psquare.py#L47>.
+fn normal_distribution_percentile(size: i32, p: f64) -> f64 {
+    let mu = 500.;
+    let sigma = 100.;
+    let normal = Normal::new(mu, sigma);
+    let mut observations = (0..size)
+        .map(|_| normal.unwrap().sample(&mut rand::thread_rng()))
+        .collect::<Vec<f64>>();
+    float_ord::sort(&mut observations);
+    let idx = ((f64::from(size) - 1.) * p) as usize;
+    observations[idx]
+}
+
+fn normal_distribution_quantile(size: i32, p: f64) -> f64 {
+    let mu = 500.;
+    let sigma = 100.;
+    let normal = Normal::new(mu, sigma);
+    let mut observations = (0..size)
+        .map(|_| normal.unwrap().sample(&mut rand::thread_rng()))
+        .collect::<Vec<f64>>();
+    float_ord::sort(&mut observations);
+
+    let mut q = Quantile::new(p).unwrap();
+    for o in observations.iter() {
+        q.append(*o).unwrap();
+    }
```

Comment:
I might be missing something, but I don't believe we need to either store or sort these observations in this case. Pretty sure we only need:

```
for i in 0..size {
    q.append(normal.sample(&mut rand::thread_rng())).unwrap();
}
```

Reply:
this was emulating what numpy percentile did under the hood, but is probably not needed.

Reply:
Oh, ok. I was just pointing out that if the goal is to benchmark the `Quantile` type, I imagine that the extra work of allocating and sorting the observations will obscure the benchmarks.

Reply:
yeah, honestly, maybe we just want the one benchmark on quantile without the comparison. the other part was mainly left in the comparison of results. Will push an update.